draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1328,6 +1328,15 @@ achieved in salted-hash based selective disclosure approaches, such as SD-JWT, a
 issued credential with the Issuer's signature is directly presented to the Verifier, who can forward it to
 the Issuer.
 
+In considering Issuer/Verifier unlinkability, it is important to note the potential for an asymmetric power dynamic
+between Issuers and Verifiers. This dynamic can compel an otherwise honest Verifier into collusion.
+For example, a governmental Issuer might have the authority to mandate that a Verifier report back information
+about the credentials presented to it. Legal requirements could further enforce this, explicitly undermining
+Issuer/Verifier unlinkability. Similarly, a large service provider issuing credentials might implicitly pressure
+Verifiers into collusion by incentivizing participation in their larger ecosystem.
+Deployments of SD-JWT must be aware of these potential power dynamics,
+mitigate them as much as possible, and/or make the risks transparent to the End-User.
```

Comment:
Not sure if deployments can be aware, so maybe...
```suggestion
Deployers of SD-JWT must be aware of these potential power dynamics,
mitigate them as much as possible, and/or make the risks transparent to the End-User.
```
(We could say "users", but that might be misleading.)

Reply:
I struggled with what word to use there. Wanting to capture like all the decision making and even implementing folks involved in a rollout/deployment of SD-JWT. "Deployments" was the best I could come up with as a general overarching term.  But you are right - deployments don't have the capacity for awareness. And "users" felt too much like end-users. I'm not seeing the *right* word exactly but your suggestion of "deployers" seems like an improvement over what I had at least.